smokey/src/main/java/frc/robot/control/AutonomousChooser.java

```diff
@@ -16,25 +16,34 @@
 import edu.wpi.first.math.geometry.Rotation2d;
 import edu.wpi.first.math.geometry.Transform2d;
 import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.Trajectory;
 import edu.wpi.first.math.trajectory.constraint.RectangularRegionConstraint;
 import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.*;
+import frc.robot.Constants;
 //import edu.wpi.first.math.geometry.Pose2d;
 //import edu.wpi.first.math.geometry.Rotation2d;
-import frc.robot.commands.DriveTimeCommand;
 import frc.robot.commands.DriveToPointCommand;
 import frc.robot.commands.DriveTrajectoryCommand;
+import frc.robot.commands.ManipulatePickerCommand;
+import frc.robot.commands.MoveArmToScoring;
+import frc.robot.commands.MoveArmToStowed;
 import frc.robot.control.Trajectories;
+import edu.wpi.first.math.util.Units;
 
 /**
  * A class for choosing different auto mode routines from shuffleboard
  */
 public class AutonomousChooser {
     private SubsystemCollection subsystems;
-    private final SendableChooser<AutonomousMode> autonomousModeChooser = new SendableChooser<>();
+    private final SendableChooser<AutonomousPath> AutonomousPathChooser = new SendableChooser<>();
+    private final SendableChooser<AutonomousBalance> balanceChooser = new SendableChooser<>();
+    private final SendableChooser<ScoringPosition> scoreHeight = new SendableChooser<>();
     private Trajectories trajectories;
-
+    
+    private Transform2d IntoNodeTransform = new Transform2d(new Translation2d(Units.inchesToMeters(Constants.snoutDepth * -1), 0), new Rotation2d(0.0));
```

Comment:
variable name should have lower case first letter: intoNodeTransform.

Reply:
done